java-api-core/src/com/kinvey/java/sync/RequestMethod.java

```diff
@@ -0,0 +1,25 @@
+package com.kinvey.java.sync;
+
+/**
+ * Created by yuliya on 09/28/17.
+ */
+
+public enum RequestMethod {
+    SAVE("SAVE"),
+    DELETE("DELETE");
+
+    private String query;
+
+    RequestMethod(String query) {
+        this.query = query;
+    }
+
+    public static RequestMethod fromString(String verb){
+        for (RequestMethod v : RequestMethod.values()){
+            if (v.query.equals(verb)){
+                return v;
+            }
+        }
+        return null;
+    }
+}
```

Comment:
Should we capture HTTP verbs instead, to make sure we are doing the appropriate action (POST, PUT, DELETE)?

Reply:
I've saved creating request logic. SDK decides which action (POST, PUT, DELETE) to use before pushing, NetworkManager#saveBlocking and NetworkManager#deleteBlocking is used for it in PushRequest and AsyncPushRequest classes.